railties/CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+*   Use Thor for `bin/rails log:clear`.
+
+    With Thor we can't run multiple commands with a single invocation, as
+    additional commands will be seen as arguments to the first command.
+
+    The following was previously possible:
+
+    ```bash
+    bin/rails log:clear tmp:clear
+    ```
+
+    After this change these should be called separately:
+
+    ```bash
+    bin/rails log:clear
+    bin/rails tmp:clear
+    ```
```

Comment:
Do you think it would be worth adding this to the Upgrading Guide too, for extra visibility?

Reply:
I've added a section to the guides.
I've tried to make it a bit more generic.